source/clear-linux/tutorials/kubernetes/kubernetes-bp.rst

```diff
@@ -0,0 +1,96 @@
+.. _kubernetes-bp:
+
+Kubernetes Best Practices on |CL|
+#################################
+
+Use swupd to update clusters
+****************************
+
+This tutorial shows you how to manage your Kubernetes cluster while using
+:command:`swupd` to update |CL-ATTR|.
+
+In our tutorial :ref:`kubernetes`, we explain how to set up a Kubernetes
+cluster on |CL| using `kubeadm`. `Kubeadm documentation`_ often builds on the
+assumption that the distribution uses a traditional package manager (e.g.,
+RPM/DEB).
+
+In contrast, |CL| uses `swupd` to update the OS, which in this case updates
+all of the kubernetes node and client binaries simultaneously, as part of
+the `cloud-native-basic` bundle (e.g., kubectl, kubeadm, kubelet). Running
+:command:`sudo swupd update` requires special care to ensure the OS
+incorporates the latest Kubernetes upgrades.
+
+This document describes best practices to manage cluster upgrades with
+`kubeadm` on a |CL|-based cluster.
+
+Prerequisites
+*************
+
+Assure that you:
+
+* Completed :ref:`kubernetes`
+* Installed the bundle `cloud-native-basic`
+
+.. note::
+
+   Other Linux\* distros shown in Kubernetes upgrade documentation reflect
+   `apt-get update`, `apt-mark hold kubeadm`, and similar commands; however, such commands **are not valid** on |CL|.
+
+Update the control plane
+************************
+
+#. Read kubernetes documentation `before you begin`_.
+
+#. On your master node, run the command:
+
+   .. code-block:: bash
+
+      sudo swupd update
+
+   .. note::
+
+      If the minor version of Kubernetes changes, |CL| shows a message-of-the-day, or `motd`.
+
+      Please note: If the `motd` appears, you **must postpone** a kubelet restart on master and nodes until the control plane is properly updated. :command:`swupd update` does not restart services automatically unless explicitly configured to do so.
+
+      Optional: If no `motd` appears, you may exit these instructions.
```

Comment:
let's drop this line after all

Reply:
@mythi, Please clarify "drop this line". Do you mean drop the entire note, or just one line?

Reply:
@mythi.  In my latest commit, I deleted the note starting on Ln 54. Please review/approve asap. Tks.

Reply:
I was referring to "Optional: If no `motd` appears, you may exit these instructions.". Sorry for the confusion.

Reply:
Aha! That's reassuring. I'm glad to hear we can still include explanation of the `motd`. Please approve latest push.